packs/data/feats.db/hellknight-dedication.json

```diff
@@ -42,12 +42,11 @@
             {
                 "key": "ActiveEffectLike",
                 "mode": "upgrade",
-                "path": "system.skills.int.rank",
+                "path": "system.skills.intimidation.rank",
```

Comment:
```suggestion
                "path": "system.skills.itm.rank",
```

Reply:
The full word works and is what we are moving towards now.